lib/libutee/tee_api_operations.c

```diff
@@ -1418,11 +1420,6 @@ TEE_Result TEE_AEEncryptFinal(TEE_OperationHandle operation,
 		goto out;
 	}
 
-	if (!srcData && !srcLen) {
-		res = TEE_SUCCESS;
-		goto out;
-	}
-
```

Comment:
This (and the similar chunk in `TEE_AEDecryptFinal()`) was added to fix a Coverity warning.
```C
static TEE_Result tee_buffer_update( /*... */)
{
	/* ... */
	if (op->buffer_offs > 0) {

		/* ... */
		/* Here src might be NULL            vvv */
		memcpy(op->buffer + op->buffer_offs, src, l);
		/* ... */
	}
	/* ... */
```